app/Modules/ExpertPanel/Models/ExpertPanel.php

```diff
@@ -249,16 +215,6 @@ public function specifications(): HasMany
         return $this->hasMany(Specification::class);
     }
 
-    /**
-     * Get all of the specifications for the ExpertPanel
-     *
-     * @return \Illuminate\Database\Eloquent\Relations\HasManyThrough
-     */
```

Comment:
This is technically a valid association, but it's not referenced in the codebase, and doesn't really belong on this model.  The caller should get the rulesets via the ExpertPanel's `specifications`.